packages/pn-personafisica-webapp/src/pages/Support.page.tsx

```diff
@@ -0,0 +1,216 @@
+type FormState = {
+  email: {
+    touched: boolean;
+    value: string;
+  };
+  confirmEmail: {
+    touched: boolean;
+    value: string;
+  };
+  errors: null | ValidationError<SupportForm>;
+};
+
+const reducer = (state: FormState, action: { type: string; payload: string }) => {
+  switch (action.type) {
+    case 'UPDATE_EMAIL':
+      return {
+        ...state,
+        email: {
+          value: action.payload,
+          touched: true,
+        },
+        errors: validator.validate({
+          email: action.payload,
+          confirmEmail: state.confirmEmail.value,
+        }),
+      };
+    case 'UPDATE_CONFIRM_EMAIL':
+      return {
+        ...state,
+        confirmEmail: {
+          value: action.payload,
+          touched: true,
+        },
+        errors: validator.validate({
+          email: state.email.value,
+          confirmEmail: action.payload,
+        }),
+      };
+    default:
+      return state;
+  }
+};
+
+const SupportPPButton: React.FC<{ children?: React.ReactNode }> = ({ children }) => {
+  const { PAGOPA_HELP_PP } = getConfiguration();
+  const handleClick = () => {
+    window.location.assign(PAGOPA_HELP_PP);
+  };
+
+  return (
+    <ButtonNaked sx={{ verticalAlign: 'top' }} onClick={handleClick}>
+      <Typography variant="body2" color="primary" display="inline">
+        {children}
+      </Typography>
+    </ButtonNaked>
+  );
+};
+
+const SupportPage: React.FC = () => {
+  const { t } = useTranslation(['support', 'common']);
+  const isMobile = useIsMobile();
+  const [formData, formDispatch] = useReducer(reducer, {
+    email: {
+      value: '',
+      touched: false,
+    },
+    confirmEmail: {
+      value: '',
+      touched: false,
+    },
+    errors: validator.validate({ email: '', confirmEmail: '' }),
+  });
+  const [zendeskAuthData, setZendeskAuthData] = useState<ZendeskAuthorizationDTO>({
+    action_url: '',
+    return_to: '',
+    jwt: '',
+  });
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+
+  const handleChange = (
+    type: 'UPDATE_EMAIL' | 'UPDATE_CONFIRM_EMAIL',
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    formDispatch({ type, payload: event.target.value });
+  };
+
+  const handleCancel = () => {
+    navigate(-1);
+  };
+
+  const handleConfirm = () => {
+    if (!formData.errors) {
+      dispatch(zendeskAuthorization(formData.email.value))
+        .unwrap()
+        .then((response) => {
+          setZendeskAuthData(response);
+        })
+        .catch(() => {});
+    }
+  };
+
+  return (
+    <Prompt
+      title={t('exit-title')}
+      message={t('exit-message')}
+      eventTrackingCallbackPromptOpened={() => {}} // impostare eventi tracking previsti
+      eventTrackingCallbackCancel={() => {}} // impostare eventi tracking previsti
+      eventTrackingCallbackConfirm={() => {}} // impostare eventi tracking previsti
+    >
+      <Grid container justifyContent="center" alignContent="center">
```

Comment:
@AndreaCimini90 I'd prefer to use a Stack over grid layout for one-dimensional layout

Reply:
@ZorbaDimatteo done